js/package.json

```
@@ -1,13 +1,21 @@
 {
   "private": true,
   "name": "@flarum/sticky",
+  "prettier": "@flarum/prettier-config",
   "dependencies": {
-    "flarum-webpack-config": "0.1.0-beta.10",
-    "webpack": "^4.43.0",
-    "webpack-cli": "^3.3.11"
+    "flarum-webpack-config": "1.0.0",
```

I think this should be:

```suggestion
    "flarum-webpack-config": "^1.0.0",
```